src/node-loaders.js

```diff
@@ -34,20 +33,26 @@ var FileSystemLoader = Loader.extend({
         }
 
         if(opts.watch) {
-            // Watch all the templates in the paths and fire an event when
-            // they change
-            var paths = this.searchPaths.filter(function(p) { return existsSync(p); });
-            var watcher = chokidar.watch(paths);
-            var _this = this;
-            watcher.on('all', function(event, fullname) {
-                fullname = path.resolve(fullname);
-                if(event === 'change' && fullname in _this.pathsToNames) {
-                    _this.emit('update', _this.pathsToNames[fullname]);
-                }
-            });
-            watcher.on('error', function(error) {
-                console.log('Watcher error: ' + error);
-            });
+            try {
+                var chokidar = require('chokidar');
+
+                // Watch all the templates in the paths and fire an event when they change
+                var paths = this.searchPaths.filter(function(p) { return existsSync(p); });
+                var watcher = chokidar.watch(paths);
+                var _this = this;
+                watcher.on('all', function(event, fullname) {
+                    fullname = path.resolve(fullname);
+                    if(event === 'change' && fullname in _this.pathsToNames) {
+                        _this.emit('update', _this.pathsToNames[fullname]);
+                    }
+                });
+                watcher.on('error', function(error) {
+                    console.log('Watcher error: ' + error);
+                });
+            }
+            catch (err) {
```

Comment:
This seems like a really broad catch, which could easily catch some other (unexpected) error, and proceed to give a wrong and misleading message on the console.

Where exactly will an error be raised if chokidar isn't installed? On the `require('chokidar')` line, right? So why not just wrap that line only?